test/.gitignore

```diff
@@ -1 +1,9 @@
 test
+
+Makefile
+CTestTestfile.cmake
+ds3_c_tests
```

Comment:
Is this a directory?  if so, the convention has been to put a / at the end of the directory name.

Reply:
ds3_c_tests is an executable

Reply:
Got it

Reply:
I did update a separate entry that was a directory